sycl/include/sycl/multi_ptr.hpp

```diff
@@ -1006,6 +1009,9 @@ class multi_ptr<void, Space, access::decorated::legacy> {
 
   static constexpr access::address_space address_space = Space;
 
+  using decorated_type =
+      typename detail::DecoratedType<const void, Space>::type;
```

Comment:
Same applies to this. The places in this class that uses it should be able to use `pointer_t` instead.

Reply:
@steffenlarsen , thanks, done.